AISKU/Tests/Perf/src/aiskuperftests.ts

```diff
@@ -0,0 +1,7 @@
+import { AISKUPerf } from "./AISKUPerf.Tests";
+//Do not remove this import, it will be used in browser.
+import { Snippet } from "../../../src/Initialization";
```

Comment:
When removing this import (this import wouldn't be used in the code), it will have `Failed to load resource: the server responded with a status of 404 (Not Found)` when processing with `Returning module - @microsoft/ai-test-framework`.